python_modules/libraries/dagster-dg/dagster_dg/cli/list.py

```diff
@@ -396,14 +445,34 @@ def list_env_command(path: Path, **global_options: object) -> None:
         click.echo("No environment variables are defined for this project.")
         return
 
+    env_var_keys = env.values.keys() | used_env_vars.keys()
+    plus_keys = _get_dagster_plus_keys(dg_context.project_name, env_var_keys)
+
     table = Table(border_style="dim")
```

Comment:
consider using `DagsterOuterTable` so that this gets the same styling as other tables